index.html

```diff
@@ -52,6 +52,9 @@
 		* [Filmic Worlds Blog](http://filmicworlds.com/blog/)
 		* [High Dynamic Range Imaging Book](https://www.cl.cam.ac.uk/~rkm38/pdfs/mantiuk15hdri.pdf)
 		* [Tone Mapping (slides)](https://www.cl.cam.ac.uk/~rkm38/pdfs/tone_mapping.pdf)
+		
+		### Research
+		* [6DOF VR experiences in iOS](http://jmdmobilesolutions.com/vr6dof-ios/)
```

Comment:
Move this up so lvl3 headings are in alphabetical order.  A better section title may be "VR". I'd also recommend touching up the style on your site before we accept.